app/components/npq_separation/navigation_structures/admin_navigation_structure.rb

```diff
@@ -13,17 +13,7 @@ def structure
             name: "Dashboard",
             href: npq_separation_admin_path,
             prefix: "/npq-separation/admin/dashboard",
-          ) => [
-            Node.new(
-              name: "Summary",
-              href: npq_separation_admin_dashboards_summary_path,
-              prefix: "/npq-separation/admin/dashboard",
-              nodes: [
-                Node.new(name: "Dashboard 1", href: "#", prefix: "/npq-separation/admin/dashboards/one"),
```

Comment:
I know this isn't in the ticket but maybe worth keeping the side navigation to go to the different tables per lead provider vs per course?

Reply:
discussed in person: leave as is